Error compiling makefile

Error
arm-none-eabi-gcc -I./ -I./spl/CMSIS -I./spl/CMSIS/inc -I./spl/inc -I./libfixmath -DSTM32F10X_MD -DUSE_STDPERIPH_DRIVER -c -fno-common -O0 -g -mcpu=cortex-m3 -mthumb -ffunction-sections -fdata-sections IMU/MPU6050/MPU6050.c -o IMU/MPU6050/MPU6050.o
In file included from IMU/MPU6050/MPU6050.h:39:0,
from IMU/MPU6050/MPU6050.c:32:
IMU/MPU6050/MPU6050.c: In function 'MPU6050_I2C_Init':
IMU/MPU6050/HAL_MPU6050.h:54:40: error: 'MPU6050_SCL__PIN' undeclared (first use in this function)
# define MPU6050_I2C_SCL_Pin MPU6050_SCL__PIN

^
IMU/MPU6050/MPU6050.c:311:33: note: in expansion of macro 'MPU6050_I2C_SCL_Pin'
GPIO_InitStructure.GPIO_Pin = MPU6050_I2C_SCL_Pin | MPU6050_I2C_SDA_Pin;
^
IMU/MPU6050/HAL_MPU6050.h:54:40: note: each undeclared identifier is reported only once for each function it appears in
# define MPU6050_I2C_SCL_Pin MPU6050_SCL__PIN

^
IMU/MPU6050/MPU6050.c:311:33: note: in expansion of macro 'MPU6050_I2C_SCL_Pin'
GPIO_InitStructure.GPIO_Pin = MPU6050_I2C_SCL_Pin | MPU6050_I2C_SDA_Pin;
^
IMU/MPU6050/HAL_MPU6050.h:55:40: error: 'MPU6050_SDA__PIN' undeclared (first use in this function)
# define MPU6050_I2C_SDA_Pin MPU6050_SDA__PIN

^
IMU/MPU6050/MPU6050.c:311:55: note: in expansion of macro 'MPU6050_I2C_SDA_Pin'
GPIO_InitStructure.GPIO_Pin = MPU6050_I2C_SCL_Pin | MPU6050_I2C_SDA_Pin;
^
IMU/MPU6050/HAL_MPU6050.h:66:38: error: 'MPU6050_SCL__PORT' undeclared (first use in this function)
# define MPU6050_I2C_Port MPU6050_SCL__PORT

^
IMU/MPU6050/MPU6050.c:314:13: note: in expansion of macro 'MPU6050_I2C_Port'
GPIO_Init(MPU6050_I2C_Port, &GPIO_InitStructure);
^
Makefile:80: recipe for target 'IMU/MPU6050/MPU6050.o' failed
make: **\* [IMU/MPU6050/MPU6050.o] Error 1

--- a/IMU/MPU6050/HAL_MPU6050.h
+++ b/IMU/MPU6050/HAL_MPU6050.h
@@ -51,8 +51,8 @@ extern "C" {
 
   #define MPU6050_I2C                  I2C1
   #define MPU6050_I2C_RCC_Periph       RCC_APB1Periph_I2C1
-  #define MPU6050_I2C_SCL_Pin          MPU6050_SCL__PIN
-  #define MPU6050_I2C_SDA_Pin          MPU6050_SDA__PIN
+  #define MPU6050_I2C_SCL_Pin          GPIO_Pin_6
+  #define MPU6050_I2C_SDA_Pin          GPIO_Pin_7
 
 #elif MPU6050_I2C_USE_I2C2
 
@@ -63,7 +63,7 @@ extern "C" {
 
 #endif
 
-#define MPU6050_I2C_Port             MPU6050_SCL__PORT
+#define MPU6050_I2C_Port             GPIOB
 #define MPU6050_I2C_RCC_Port         RCC_APB2Periph_GPIOB
 #define MPU6050_I2C_Speed            100000 // 100kHz standard mode
 

--- a/gpio.h
+++ b/gpio.h
@@ -50,7 +50,8 @@
 #ifndef GPIO_H
 #define GPIO_H
 
-#define CURRENT_PHASE_A__PIN      GPIO_Pin_3#define LED_1_BATTERY_INDICATOR__PIN        GPIO_Pin_15
+#define CURRENT_PHASE_A__PIN      GPIO_Pin_3
+#define LED_1_BATTERY_INDICATOR__PIN        GPIO_Pin_15
 #define LED_1_BATTERY_INDICATOR__PORT       GPIOB
 #define CURRENT_PHASE_A__PORT     GPIOA
 #define CURRENT_PHASE_C__PIN      GPIO_Pin_2

